app/controllers/hxlstats_controller.rb

```diff
@@ -0,0 +1,69 @@
+class HxlstatsController < ApplicationController
+  # Class to collect and aggregate data to parse to the HXL proxy/endpoint
+  def new
+  end
+
+  def create
+  end
+
+  # rubocop:disable MethodLength
+  def show
+    # Prepare output results array
+    @results = []
+    @results.append(["Emotional State", "Stage of journey", "Country drawn in", "Total children affected",
```

Comment:
As well as these headings, we need another row with the HXL tagged headings (e.g. `#impact +indicator +code`) as per the spreadsheet. Both the headings and HXL headings can probably live as constants in the HxlStatsView model, or service object as suggested above.